src/config/dynamic-theme-fixes.config

```
@@ -12202,6 +12202,7 @@ todoist.com
 
 INVERT
 .ist_button_apple > img
+path[d^="M166.805 16.694"]
```

What about something like:
```suggestion
#td-help-logo_svg__a ~ g:last-of-type
```